src/main/java/org/datadog/jmxfetch/JMXAttribute.java

```diff
@@ -31,9 +31,9 @@
     private static final String DOT_UNDERSCORE = "_*\\._*";
     private MBeanAttributeInfo attribute;
     private Connection connection;
-    private ObjectInstance jmxInstance;
+    private ObjectName jmxInstance;
```

Comment:
Could you rename this attribute to `beanName` like you did in `JMXSimpleAttribute` and `JMXComplexAttribute` ? :smiley: